tests/contrib/django_util/test_django_util.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+
+import django.conf
+from django.conf.urls import include, url
+from django.contrib.auth.models import AnonymousUser
+from django.core import exceptions
+import mock
+from six.moves import reload_module
+from tests.contrib.django_util import TestWithDjangoEnvironment
+import unittest2
+
+from oauth2client.contrib import django_util
+import oauth2client.contrib.django_util
+from oauth2client.contrib.django_util import (
+    _CREDENTIALS_KEY, get_storage, site, UserOAuth2)
+
+
+urlpatterns = [
+    url(r'^oauth2/', include(site.urls))
+]
+
+urlpatterns += [url(r'^oauth2/', include(site.urls))]
+
+
+class OAuth2SetupTest(unittest2.TestCase):
+
+    def setUp(self):
+        import django.conf
```

Comment:
Think I cleaned up all occurrences of this, it was unnecessary, was guessing-and-checking django setup stuff a bit and was doing silly things.